app/models/team_member.rb

```diff
@@ -38,4 +38,12 @@ def job_title
       discipline
     end
   end
+
+  def delivery_first
+    discipline == "Delivery" ? 0 : 1
+  end
+
+  def self.delivery_first(team_members)
```

Comment:
I think maybe a more intention-revealing name for this method would be good here. Something like `sorted_with_delivery_team_members_first`?

Reply:
I'm trying to find a balance between "descriptive" and "as soon as we install rubocop it's going to scream about line too long". 🙈